bin/ci/clone-gecko.sh

```diff
@@ -1,12 +1,9 @@
 #!/bin/bash
 
 hg --version
-if [ ! -d "gecko/.hg" ]; then
-  rm -rf gecko/
-  hg clone https://hg.mozilla.org/mozilla-unified/ gecko
-else
-  cd gecko
-  hg strip --no-backup 'roots(outgoing())'
-  hg pull -u
-  cd ..
-fi
+rm -rf firefox/
+hg clone https://hg.mozilla.org/mozilla-unified/ firefox
+
+cd firefox
+hg co $MC_COMMIT
+cd ..
```

Comment:
It could be great to use `bin/prepare-mochitests-dev`
so that CI and contributors go through the same path.

But given this change, it looks like you had issue with keeping m-c clone around between two travis runs?

Reply:
yeah, caching was an issue.